app-config.yaml

```diff
@@ -237,9 +237,12 @@ costInsights:
       icon: search
   metrics:
     dailyCost:
-      name: Your Company's Daily Cost
+      name: Pied Piper's Daily Cost
+      compare: DAU
```

Comment:
Hm, are we setting `compare` to get the default comparison metric ? Maybe we can just get the first metric from the list ? From first glance, I would assume that the `compare` metric is the only available option for comparison, which isn't true WDYT?

Reply:
> Maybe we can just get the first metric from the list ?

I considered this approach initially. Couple thoughts:
1. Metrics as a whole are optional, so there isn't necessarily a guarantee that another metric would exist. `dailyCost` itself is optional, so a user could define some other metric "first" and we'd have to handle that condition (which isn't terrible but just more complexity) - is that "first" metric a default comparison or not? What if the user doesn't want it to be the default comparison?
2. Somewhat minor/philosophical: The metric field is a dict, so in theory at least, properties are not "ordered" in the same way as with a list/array/etc. So imparting significance on property order is a bit weird and uncommon in configurations.

> From first glance, I would assume that the compare metric is the only available option for comparison, which isn't true WDYT?

Good point. I think this could be improved.

Reply:
What about adding `default: true` to the metric definition? We'd need to validate that only one metric had this property, but it seems slightly more natural than adding a metric name to `dailyCost`.

Reply:
Yeah, makes sense. I feel like this alongside the broader API changes would lend itself to removing `dailyCost` entirely from the `metric` field in configuration. See my comments below.